index.html

```diff
@@ -183,6 +183,47 @@ <h4 class="skill-name">Skills</h4>
                 </ul>
             </div>
         </section>
+        <section class="get-in-touch">
+            <div class="invitation">
+                <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+            </div>
+            <form class="form"  action="https://formspree.io/f/xjvdqolb"
+            method="POST">
```

Comment:
- [ ] Hey @team, check this out, your form is not active as says this error, please fix the issue, make your form spree submitting fine, and request another review

<img width="379" alt="1" src="https://github.com/Kccendana/portfolio_project/assets/77843050/8e242078-03d8-4aa0-ab39-efd866a3b97b">